### PR TITLE
Documentation link updates

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Pull request checklist
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Updating this links because I am pulling the markdown files shipped by OUFR package into generating an Overview page for each control on docs.microsoft portal like [here](https://review.docs.microsoft.com/en-us/javascript/docs/overviews/buttonoverview?view=office-ui-fabric-react-latest&branch=testPackageSetup) and need those links to have valid hrefs so that a user can navigate to Fabric website.

@ecraig12345 I didn't dig deep into the urls handling by the fabric-website so maybe you could let me know if this could break something? Thanks!

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9441)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Some real-world applications of the component include copying, uploading, or mov
 
 The Announced component currently has the following documented use cases:
 
-1. [Quick Actions](#/controls/web/announced/quickactions): Operations such as editing text or deletion that are short enough that they do not require a status during progress.
-2. [Search Results](#/controls/web/announced/searchresults): Appearance of search results such as in contact fields or search boxes.
-3. [Lazy Loading](#/controls/web/announced/lazyloading): "Lazy loading" of page sections that do not appear all at once.
-4. [Bulk Operations](#/controls/web/announced/bulkoperations): Operations that require multiple sub operations, such as the moving of several files.
+1. [Quick Actions](https://developer.microsoft.com/en-us/fabric#/controls/web/announced/quickactions): Operations such as editing text or deletion that are short enough that they do not require a status during progress.
```
Comment (ecraig12345) [inline packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md]: Are the hash links not working in some particular context?

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md]: Yes. We basically are using the already existing markdown files that are shipped with OUFR package for the HIG website. In the context of the website those links are working OK, but on docs.microsoft portal they don't make sense as it just appends it to the url.

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/Announced/docs/AnnouncedOverview.md]: I have a link in the description of this PR where if you click the `IIconProps` link it does nothing

**Comment by Vitalius1** [comment]
looking into some other solution because this might bring some user experience regression

**Comment by jordandrako** [comment]
Yeah, another solution will be needed here, as this will break links on the HIG for pages it imports. Right now it imports all the styles and controls pages, so those need to be hash links.

**Comment by Vitalius1** [comment]
@jordandrako @ecraig12345 I've updated this and only removed some unnecessary links

**Pull request merged**